apps/system/style/statusbar/statusbar.css

```diff
@@ -25,9 +25,12 @@
   pointer-events: auto;
 }
 
-#statusbar.hidden {
-  opacity: 0;
-  visibility: hidden;
+
+/* Hide the statusbar out of the viewport in order
+to be available for moz-elements */
+#statusbar.hidden,
+#screen.utility-tray-in-transition > #statusbar {
+  top: calc(var(--statusbar-height) * -1);
```

Comment:
should've caught this during the first round.
but currently some of the events toggling the `.hidden` expect it not to cause any reflow (especially for edge gestures)

can we `transform:translate(-100%)` it instead of is it an accessibility issue?